README.md

```diff
@@ -10,6 +10,15 @@ Spoke was created by Saikat Chakrabarti and Sheena Pakanati, and is now maintain
 
 This is generated from [react-apollo-starter-kit](https://github.com/saikat/react-apollo-starter-kit).  Look at that project's README for info on some of the libraries used.
 
+## Deploy to Heroku
+
+<a href="https://heroku.com/deploy?template=https://github.com/shakalee14/Spoke">
```

Comment:
change this to `MoveOnOrg` from shakalee :-)

Reply:
Also, we should have a warning here that this template incurs costs and is not supported on a free instance/version (or if it can, we should say that too)